.github/workflows/tests.yml

```diff
@@ -19,21 +19,30 @@ jobs:
     # image: The Docker image to use.
     #
     # pre-checkout-setup: Commands to run before the git repo checkout.
-    #   If git is not in the Docker image, it must be installed here.
-    #   Otherwise, the checkout action uses the GitHub REST API, which
-    #   doesn't result in an actual git repo. A real git repo is
-    #   required to checkout the submodules.
+    #   If bash or git are not in the Docker image, they must be
+    #   installed here. The `ci` scripts use bash, so they'll fail
+    #   immediately without it. Without git, the checkout action uses
+    #   the GitHub REST API, which doesn't result in an actual git repo.
+    #   A real git repo is required to checkout the submodules.
     #
     # extra-packages: Packages to install in addition to those in
     #   `ci/gh-install.sh`. This can be used to support features from
     #   additional `configure` options.
     #
     # configure-options: Options to pass to `configure`.
     strategy:
+      # Let other configurations continue if one fails.
+      fail-fast: false
+
       matrix:
         include:
-          - name: Debian Buster (10) with sign-ed25519
-            image: debian:buster-slim
+          # Debian builds. Currently stable and testing are tested.
+          # Other options would be stable-backports, oldstable,
+          # oldstable-backports and unstable.
```

Comment:
I don't know how the Docker images are set up, but as general information: -backports is an overlay (like an Ubuntu PPA or Fedora COPR) rather than a fully independent version, and packages are not installed from -backports by default, even there's a -backports apt source configured in `/etc/apt/sources.list` or `/etc/apt/sources.list.d/`. If you want to install packages from -backports you have to use something like `apt-get -t stable-backports install libfoo1`.

Reply:
Good point. Taking a closer look at `stable-backports`, it just has the stable source in `/etc/apt/sources.list.d` with no other special configuration. So, it would just use the default apt preferences that downgrades the priority since the `Release` file has `NotAutomatic: yes`.

I wasn't planning to add builds for those, so I could just remove the comments here to just mention `oldstable` and `unstable`.